File: BEE 1015.cpp
#include<iostream>
#include<math.h>
#include<iomanip>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(4) << (x)

using namespace std;

int main(){
    float x1, x2, y1, y2;
    cin >> x1 >> y1;
    cin >> x2 >> y2;
    float distance = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
    cout << FIXED_FLOAT(distance) << endl;
    return 0;
}

File: BEE 1015.php
<?php

    list($x1, $y1) = explode(" ", readline());
    list($x2, $y2) = explode(" ", readline());
    $x1 = (float) $x1;
    $y1 = (float) $y1;
    $x2 = (float) $x2;
    $y2 = (float) $y2;
    $distance = sqrt(pow($x2 -$x1, 2) + pow($y2 - $y1, 2));
    $distance = number_format($distance, 4, ".", "");
    echo "$distance\n";

?>